javascript/javascript_in_the_real_world/linting.md: added info about the eslint and prettier conflict when used together and a fix (#24721)
TheOdinProject/curriculum: javascript/javascript_in_the_real_world/linting.md

Using prettier makes coding faster and easier! You don't have to worry about nailing things like indentation, or remembering every semi-colon because prettier will take care of those details for you.

### Note on using Eslint and Prettier together

> Using eslint and prettier together causes conflicts. To fix that follow the instructions to install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation). It turns off all eslint rules that are unnecessary or might conflict with Prettier. Doing just this is enough to resolev the conflict and get them both working smoothly with one another.

Reviewer: ```suggestion
Using ESLint and Prettier together causes conflicts. To fix that follow the instructions to install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier#installation). It turns off all ESLint rules that are unnecessary or might conflict with Prettier. Doing just this is enough to resolve the conflict and get them both working smoothly with one another.
```
There's a typo, as well as fixing capitalization on Prettier and ESLint.

> Another way to address the conflict is to use `eslint-plugin-prettier`. It lets you run prettier as if it was a linter rule. However, doing this is **not recommended**. You can learn more about it [here](https://prettier.io/docs/en/integrating-with-linters.html#notes)

Reviewer: ```suggestion
Another way to address the conflict is to use `eslint-plugin-prettier`. It lets you run Prettier as if it were a rule in ESLint. However, doing this is **not recommended**. You can learn more about it [here](https://prettier.io/docs/en/integrating-with-linters.html#notes)
```
Fixing capitalization and changing up sentence structure a bit.
